backend/app/usecase/shortlink/updater_test.go

```diff
@@ -238,8 +329,8 @@ func TestShortLinkUpdaterPersist_UpdateShortLink(t *testing.T) {
 				ID:    "1",
 				Email: "gopher@golang.org",
 			},
-			update: entity.ShortLink{
-				LongLink: "http://malware.wicar.org/data/ms14_064_ole_not_xp.html",
+			shortLinkInput: entity.ShortLinkInput{
```

Comment:
"reject malicious long link" now fails. There seems to be no error coming out of it when it should. I will investigate and resolve this comment when it's fixed.

Reply:
Fixed. It was using the old long link reference.